core/source_generator/source_generator.cc

```diff
@@ -71,13 +71,22 @@ string prettySigForMethod(const core::GlobalState &gs, core::MethodRef method, c
     if (!flags.empty()) {
         flagString = fmt::format("{}.", fmt::join(flags, "."));
     }
+    string typeParamString = "";
+    if (!sym->typeArguments().empty()) {
+        for (const auto &typeParam : sym->typeArguments()) {
+            typeParams.push_back(fmt::format(":{}", typeParam.data(gs)->name.show(gs)));
+        }
+        typeParamString = fmt::format("type_parameters({}).", fmt::join(typeParams, ", "));
```

Comment:
We have a helper to make this a little more concise (and slightly more performant). Search for `fmt::map_join` in this repo.